crates/notion-core/src/event.rs

```diff
@@ -121,11 +122,15 @@ impl EventLog {
         self.events.push(event);
     }
 
-    // send the events from this session to the monitor
-    pub fn send_events(&mut self, command: Option<String>) {
-        self.monitor
-            .get_mut(command)
-            .unwrap()
-            .send_events(&self.events);
+    pub fn publish(&mut self, plugin: Option<&Publish>) {
+        match plugin {
+            Some(&Publish::Url(_)) => { unimplemented!() }
+            Some(&Publish::Bin(ref command)) => {
```

Comment:
I like the way this is setup now, but in the future I think it would be nice to support sending events to multiple places, for instance logging to a file as well as sending to a URL, to be more flexible.

Reply:
Good point. Also at some point it needs some better data abstraction one way or another; there are too many types that are `Foo::{Url, Bin}`. I filed #113.